spark/src/test/java/io/spark/ddf/etl/TransformationHandlerTest.java

```diff
@@ -88,7 +88,7 @@ public void testReservedFactor() throws DDFException {
 
     Assert.assertEquals(31, ddf.getNumRows());
     Assert.assertEquals(9, ddf.getNumColumns());
-    Assert.assertEquals("test123", ddf.getColumnName(8));
+    Assert.assertEquals("test123", ddf.getColumnName(0));
     Assert.assertEquals(9, ddf.VIEWS.head(1).get(0).split("\\t").length);
```

Comment:
@nhanitvn , yes I discussed with Bao actually. I didn't think of the R transform part. The reason I left the columns as they are is because I don't want to introduce more latency(tiny ones add up). The logic right now is a bit convoluted because the API and the behavior we want to support is complicated. We can add/select/delete/redefine columns in the RExp, and we can also select columns in another parameter. Let me re-think and see if I can improve this.